_posts/2020-07-12-changelog-126.markdown

```diff
@@ -0,0 +1,113 @@
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5687 -->
+* Add an option to sort items alphabetically in `Structure View` (<kbd>Ctrl + F12</kbd>) (by [@t-kameyama])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5652 -->
+* Allow extracting non-root modules in `Extract inline module` intention action (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5705 -->
+* UI: update icons for _targets_ in the Cargo toolwindow
+ (see [differences](https://github.com/intellij-rust/intellij-rust/pull/5705))
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5693 -->
+* Notify user if it uses too old Rust toolchain that is unsupported by IntelliJ Rust.
+ Today we support Rust `1.32.0` or higher.
+
+## Fixes
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5612 -->
+* Use more precise error annotations for missing/extraneous function call arguments.
+ Previously, all function arguments were highlighted (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5653 -->
+* Render aliased types for let bindings in the quick documentation <kbd>Ctrl + Q</kbd> (by [@Kobzol])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5689 -->
+* Fix finding location of `evcxr` (Rust REPL) and `grcov` (needed for `Run with Coverage`) on arch linux
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5683 -->
+* Fix `Run external linter` action (was broken in the previous release) (by [@Stzx])
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5679 -->
+* Fix infinite cargo refresh if you have `openid = 0.4.0` dependency crate
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5678 -->
+* Fix false-positive `Lift return` inspection on empty match expr
+ See [the issue](https://github.com/intellij-rust/intellij-rust/issues/5677) for details.
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5670 -->
+* Fix some false-positives in move analysis related to path assignments.
+ See [the issue](https://github.com/intellij-rust/intellij-rust/issues/5394) for details.
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5669 -->
+* Fix some corner cases in doc rendering (<kbd>Ctrl + Q</kbd>).
+ See [the issue](https://github.com/intellij-rust/intellij-rust/issues/5631) for details.
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5441 -->
+* Fix `Run with Coverage`
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5701 -->
+* Test Tool Window now can be disabled by in registry: <kbd>Ctrl + Shift + A</kbd> -> type "Registry..." ->
+ `org.rust.cargo.test.tool.window`. (The registry key was removed in the previous release).
+ See [the issue](https://github.com/intellij-rust/intellij-rust/issues/5676) for details.
+
+## Performance Improvements
+
+<!-- https://github.com/intellij-rust/intellij-rust/pull/5674 -->
+* Slightly optimize `RsModulesIndex`: it should slightly speed up highlighting and macro expansion
```

Comment:
Do we really want to write about `RsModulesIndex`? Maybe "Slightly speed up highlighting and macro expansion" is enough?